src/CompositeRepository.php

```diff
@@ -235,7 +269,9 @@ private function splitPath($path)
                     return array($mountPoint, $path);
                 }
 
-                return array($mountPoint, substr($path, strlen($mountPoint)));
+                $subPath = substr($path, strlen($mountPoint)) ? : '/';
```

Comment:
This should probably be in a different PR. It fixes the issue when trying to access the root resource of composite respources.